config/config.js

```diff
@@ -28,8 +28,32 @@ const config = {
         database: 'test'
     },
 
+    VALIDATOR: {
+        /**
+         * Add custom validators here
+         * Format: { "validatorName": function(value, [additional arguments]), ... }
+         */
+        customValidators: {
```

Comment:
I thought they'd have a lot more validators built-in... =\ Looks like we need to customize our own validators? That's kinda annoying...